store/tikv/lock_resolver.go

```diff
@@ -146,6 +149,79 @@ func (lr *LockResolver) getResolved(txnID uint64) (TxnStatus, bool) {
 	return s, ok
 }
 
+// BatchResolveLocks resolve locks in a batch
+func (lr *LockResolver) BatchResolveLocks(bo *Backoffer, locks []*Lock, loc RegionVerID) (bool, error) {
+	if len(locks) == 0 {
+		return true, nil
+	}
+
+	lockResolverCounter.WithLabelValues("resolve").Inc()
+
+	var expiredLocks []*Lock
+	for _, l := range locks {
+		if lr.store.GetOracle().IsExpired(l.TxnID, l.TTL) {
+			lockResolverCounter.WithLabelValues("expired").Inc()
+			expiredLocks = append(expiredLocks, l)
+		} else {
+			lockResolverCounter.WithLabelValues("not_expired").Inc()
+		}
+	}
+	if len(expiredLocks) == 0 {
+		log.Error("This should not occur!")
+		return true, nil
+	}
+
+	lr.cntTxnStatus = 0
+	lr.costTxnStatus = 0
+	txnInfos := make(map[uint64]uint64)
+	count := 0
+	for _, l := range expiredLocks {
+		count = count + 1
+		status, err := lr.getTxnStatus(bo, l.TxnID, l.Primary)
+		if err != nil {
+			return false, errors.Trace(err)
+		}
+		txnInfos[l.TxnID] = uint64(status)
+	}
+	log.Infof("BatchResolveLocks: it takes %v cost to lookup %v txn status from %v txn status", lr.costTxnStatus, lr.cntTxnStatus, count)
```

Comment:
it takes %v to lookup

Reply:
What is the difference between `cntTxnStats` and `count`?

Reply:
ok

Reply:
I have just deleted `count` here.